web/src/components/Games.jsx

```diff
@@ -73,7 +73,8 @@ export default function Games({ loading, games, saveGame, saveQuestion, saveNewG
     setSortType(event.target.value);
   };
 
-  const renderGames = () => {
+  const renderGames = (loading) => {
+    if (loading) return <Typography gutterBottom>Loading...</Typography>;
```

Comment:
Pass the loading boolean into the game render method, and show a loading message rather than the "No results found." message while the games are loading.